examples/basic_cxx_qt_qml_plugin/src/sub.rs

```diff
@@ -1,39 +0,0 @@
-// SPDX-FileCopyrightText: 2021 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
```

Comment:
Why is this file not needed anymore?

Reply:
The `basic_cxx_qt_qml` and `basic_cxx_qt_qml_plugin` used to have exactly the same tests, it seems pointless to test the same thing twice? So i've dropped that duplication here as well.

Reply:
Ah, alright, that explains it.